### DISCUSSION TITLE
Model jumps up on default AR mode (Android, Chrome latest)

### DISCUSSION THREAD

**Discussion by CE-craft** [google/model-viewer#4355] [Q&A]
Hey,

I am encountering this issue where the model jumps up after the AR session loads and the model stabilizes on a surface once the plane is detected.
If I move the phone up or look around the model jumps up, and either show up mid air or snaps to another higher surface like a shelf for example.

This is my set up and I am using Annotations as well and they change camera views on click like the 3ed example on this page
https://modelviewer.dev/examples/annotations/index.html

Android - Chrome latest

```
    <model-viewer
      class="viewer"
      src="/model.glb"
      ios-src="/iOS-model.usdz"
      shadow-intensity="1"
      interpolation-decay="200"
      ar
      ar-placement="floor"
      camera-controls
      interaction-prompt="none"
      alt="A 3D model"
      camera-orbit="56.54deg 91.91deg 84.69m"
      field-of-view="55deg"
      min-field-of-view="1deg"
      data-orbit="56.54deg 91.91deg 84.69m"
      data-target="3.87m 0.71m -0.13m"
    >
```

Thank you

**Comment by elalish** [comment]
Can you share a screen recording? I'm tempted to say this is working as intended, but a video would help to make sure.

**Reply by CE-craft** [reply]
Hi thank you so much for the fast response, I was looking around to figure out what's going on, turns out there is something wrong with the model I can't share the model because it's for a client, but when I tested with the default astronaut model everything worked just fine.

Is there anything specific settings I need to consider when exporting models for model viewer? maybe some dimensions normalization or bounding box settings or model origin ?

**Reply by elalish** [reply]
glTF specifies that all files use units of meters - a common mistake is to not account for units on export. Scale is critical for AR. There are also some gotchas if you use animations - search through our discussions if that applies to you.

**Reply by CE-craft** [reply]
Video attached below

**Comment by CE-craft** [comment]
Thank you I will, I managed to create the bug again with the default model it happens when the detected plane is low in this case the floor and move back away from the model in AR it causes it to jump up for some reason 

https://youtube.com/shorts/GGsqVnauiKU?feature=share

**Reply by elalish** [reply]
Ah yes, this is working as intended. It jumped up to the floor once the floor was detected (the astronaut is 2 meters tall). Before it found the floor, it was placed well below the floor - we place it arbitrarily at first so the experience isn't completely broken if the floor can't be found (which is kind of often).
